src/firewall/nft.rs

```diff
@@ -1288,3 +1292,28 @@ fn get_chain(base_rules: &schema::Nftables, chain: &str) -> Option<schema::Chain
 
     None
 }
+
+fn get_netavark_rules() -> Result<schema::Nftables, helper::NftablesError> {
+    match helper::get_current_ruleset(None, Some(vec!["list", "table", "inet", TABLENAME])) {
```

Comment:
any value in filing an issue upstream with nftables to see if they can give us a better API? I suppose we could also contribute one.